playpen/ansible/roles/dev/files/bashrc

```diff
@@ -75,6 +75,7 @@ _paction() {
 psmash() {
     workon pulp-smash;
     prestart;
+    make lint || return 1;
```

Comment:
returning 1 here prevents  the later call to deactivate

I recommend something like

```bash
if make lint
then
    python -m unittest2 discover ~/devel/pulp-smash/pulp_smash;
fi
deactivate
```